qiskit/pulse/commands/sample_pulse.py

```diff
@@ -80,9 +80,10 @@ def _clip(self, samples: np.ndarray, epsilon: float = 1e-5):
             clipped_samples = np.exp(1j*np.angle(samples[clip_where]), dtype=np.complex_)
             samples[clip_where] = clipped_samples
             samples_norm[clip_where] = np.abs(clipped_samples)
-
+            if np.any(samples_norm > 1.):
+                raise PulseError('inner: Pulse contains sample with norm greater than 1+epsilon.')
```

Comment:
`inner` and `outer` isn't that meaningful to a user, maybe for `inner` it can say something like `'Unable to clip samples with norm near 1 down to 1.'`